app/assets/javascripts/admin/products/services/unit_prices.js.coffee

```diff
@@ -0,0 +1,25 @@
+angular.module("admin.products").factory "UnitPrices", (VariantUnitManager) ->
+  class UnitPrices
```

Comment:
Where is this class going to be used? Can't we avoid all this client-side code completely if we're already computing and serializing the values on the server side?

Reply:
As far as I understand, this could be really useful in the backoffice part, like the new product form page : #6494

Reply:
@Matt-Yorkley I believe we want to be able to compute it immediately in the frontend (the backoffice frontend, that is) before the variant is necessarily saved.